tools/wasp-cli/decode/cmd.go

```diff
@@ -51,3 +56,16 @@ var decodeCmd = &cobra.Command{
 		}
 	},
 }
+
+var translateMetadataCmd = &cobra.Command{
+	Use:   "decode-metadata <0x...>",
+	Short: "Translates metadata from Hex to a humanly-readable format",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		metadata, err := isc.RequestMetadataFromBytes(hexutil.MustDecode(args[0]))
+		log.Check(err)
+		jsonBytes, err := json.MarshalIndent(metadata, "", " ")
```

Comment:
We always use two spaces, it's a little bit more human readable.
```suggestion
		jsonBytes, err := json.MarshalIndent(metadata, "", "  ")
```